crates/top/re_sdk/src/web_viewer.rs

```diff
@@ -11,22 +10,25 @@ pub enum WebViewerSinkError {
     #[error(transparent)]
     WebViewerServer(#[from] WebViewerServerError),
 
-    /// Failure to host the Rerun WebSocket server.
+    /// Invalid host IP.
     #[error(transparent)]
-    RerunServer(#[from] re_ws_comms::RerunServerError),
+    InvalidAddress(#[from] std::net::AddrParseError),
 }
 
 /// A [`crate::sink::LogSink`] tied to a hosted Rerun web viewer. This internally stores two servers:
-/// * A [`re_ws_comms::RerunServer`] to relay messages from the sink to a websocket connection
+/// * A gRPC server to relay messages from the sink to any connected web viewers
 /// * A [`WebViewerServer`] to serve the Wasm+HTML
 struct WebViewerSink {
     open_browser: bool,
 
-    /// Sender to send messages to the [`re_ws_comms::RerunServer`]
+    /// Sender to send messages to the gRPC server.
     sender: re_smart_channel::Sender<LogMsg>,
 
+    /// The gRPC server thread.
+    _server_handle: std::thread::JoinHandle<()>,
+
     /// Rerun websocket server.
-    rerun_server: RerunServer,
+    server_signal: re_grpc_server::shutdown::Signal,
```

Comment:
```suggestion
    server_shutdown_signal: re_grpc_server::shutdown::Signal,
```
help make it clear this isn't some general signal